src/Describer.php

```diff
@@ -83,8 +83,15 @@ protected function describeCommands(Application $application, OutputInterface $o
     {
         $this->width = 0;
 
+        $muted = collect(config('laravel-console-summary.muted_commands'));
```

Comment:
```suggestion
        $hide = collect(config('laravel-console-summary.hide'));
```

Reply:
What happens if the config don't exist?

Reply:
From the Service Provider:

`$this->mergeConfigFrom( __DIR__ . '/../config/config.php', 'laravel-console-summary' );`